airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractRelationalDbSource.java

```diff
@@ -38,6 +38,20 @@ public AutoCloseableIterator<JsonNode> queryTableFullRefresh(final Database data
         getFullTableName(schemaName, tableName)));
   }
 
+  @Override
+  public AutoCloseableIterator<JsonNode> queryTableFullRefresh(final Database database,
+                                                               final List<String> columnNames,
+                                                               final String schemaName,
+                                                               final String tableName,
+                                                               final String cursorField) {
+    LOGGER.info("Queueing query for table: {}", tableName);
+    String order = String.format("ORDER BY %s", getIdentifierWithQuoting(cursorField));
+    return queryTable(database, String.format("SELECT %s FROM %s %s",
```

Comment:
Potentially need to be careful here, this could be changing the SQL for any connector that doesnt override this. Some SQL databases follow different ORDER BY syntax. Maybe `getIdentifierWithQuoting` abstraction already takes care of this but would be important to confirm

Reply:
That's why I created the overloaded function, but maybe I missed something